src/net/ishchenko/idea/nginx/KeywordsFromSourcesGenerator.java

```diff
@@ -31,11 +31,11 @@
 
     private static final Pattern PREPROCESSOR_DIRECTIVE_PATTERN = Pattern.compile("#.+");
 
-    private static final Pattern DIRECTIVE_BLOCK_PATTERN = Pattern.compile("static\\s+ngx_command_t\\s+[^\\[]+\\[\\]\\s*=\\s*(\\{[^;]+);");

```

Comment:
I'm not familiar with this regex, can you explain what this fixes?

Reply:
It's redundant character escape '\\]' in regexp and it has the same function as before.